examples/TargetingConsoleApp/CartOptions.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+namespace Consoto.Banking.HelpDesk
+{
+    class CartOptions
+    {
+        public int Size { get; set; }
+    }
```

Comment:
Do we need 'Color' option?

Reply:
Yes, it should be added for completeness of the sample.